packages/datetime/src/dateRangeInput.tsx

```diff
@@ -111,36 +113,40 @@ export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {
 
 export interface IDateRangeInputState {
     isOpen?: boolean;
+    preferredBoundaryToModify?: DateRangeBoundary;
```

Comment:
That's what I called it in the prototype implementation, but I wanted to communicate clearly that it wouldn't _always_ be respected, hence the addition of "preferred". But if we're cool letting the prop description alone clarify that, I'm cool with calling the prop `boundaryToModify`.